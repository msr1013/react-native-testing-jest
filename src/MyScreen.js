import React from 'react';
import {View} from 'react-native';

const MyScreen = () => {
  const style = {backgroundColor: 'black'};
  return <View style={style} />;
};

export default MyScreen;
