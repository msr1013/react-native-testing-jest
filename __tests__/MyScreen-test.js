import React from 'react';
import {create} from 'react-test-renderer';
import MyScreen from '../src/MyScreen';

const tree = create(<MyScreen />);

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});
