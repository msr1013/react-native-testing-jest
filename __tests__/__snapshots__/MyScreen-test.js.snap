// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
    }
  }
/>
`;
